dev/.env.micropowermanager-backend

```diff
@@ -22,3 +22,6 @@ REDIS_PASSWORD=null
 JWT_SECRET=TihQ232bmMermM8HKaLHwzRl5JSuxwqhfEAQKxRHrQqjRUwrsw1FRZhAcLkihEDU
 
 CACHE_DRIVER=redis
+# Mailer configuration for local development
+MAIL_LOG_CHANNEL=stdout
+MAIL_MAILER=log
```

Comment:
Nit: Can you add back the empty need line at the end 😅

Reply:
> Can we can rid of MailHelperInterface entirely?

From what I understand, having a facade (wrapper) like this is what makes it easy to easily swap things when we need to. For example, during tests we can have a simplified implementation.

Refactoring from PHPMailer was straightforward because of the simple interface. 😄

Reply:
I can't think of a good enough argument now, especially when we haven't started considering the context of tests. For now I will get rid of it; we can always bring it back in the future when the use case makes more sense.

Reply:
Updated.